src/components/content/order/migrate/MigrateServiceStatusPolling.tsx

```diff
@@ -3,131 +3,132 @@
-import { useServiceDetailsPollingQuery } from '../orderStatus/useServiceDetailsPollingQuery';
 import { MigrationStatus } from '../types/MigrationStatus';
+import { useMigrateServiceDetailsPollingQuery, useServiceDetailsPollingQuery } from './useMigrateServiceQuery';
 import { OperationType } from '../types/OperationType';
+import { MigrationOrderSubmitResult } from './MigrationOrderSubmitResult';
+import { MigrationProcessingStatus } from './MigrationProcessingStatus';
 
 function MigrateServiceStatusPolling({
-    destroyUuid,
-    deployUuid,
-    isMigrateSuccess,
-    error,
-    isLoading,
+    migrationId,
+    isMigrateRequestSuccess,
+    migrateRequestError,
+    isMigrateRequestLoading,
     setIsMigrating,
-    setRequestSubmitted,
+    setMigrateDisable,
     setIsPreviousDisabled,
     getCurrentMigrationStepStatus,
     serviceHostingType,
 }: {
-    destroyUuid: string | undefined;
-    deployUuid: string | undefined;
-    isMigrateSuccess: boolean;
-    error: Error | null;
-    isLoading: boolean;
+    migrationId: string | undefined;
+    isMigrateRequestSuccess: boolean;
+    migrateRequestError: Error | null;
+    isMigrateRequestLoading: boolean;
     setIsMigrating: (arg: boolean) => void;
-    setRequestSubmitted: (arg: boolean) => void;
+    setMigrateDisable: (arg: boolean) => void;
     setIsPreviousDisabled: (arg: boolean) => void;
     getCurrentMigrationStepStatus: (srg: MigrationStatus) => void;
     serviceHostingType: DeployedServiceDetails.serviceHostingType;
 }): React.JSX.Element {
-    const [isStart, setIsStart] = useState<boolean>(false);
-
-    const getDestroyServiceEntityByIdQuery = useServiceDetailsPollingQuery(destroyUuid, isStart, serviceHostingType, [
-        DeployedServiceDetails.serviceDeploymentState.DESTROY_SUCCESSFUL,
-        DeployedServiceDetails.serviceDeploymentState.DESTROY_FAILED,
-    ]);
-
-    const getDeployServiceEntityByIdQuery = useServiceDetailsPollingQuery(deployUuid, isStart, serviceHostingType, [
-        DeployedServiceDetails.serviceDeploymentState.DEPLOYMENT_SUCCESSFUL,
-        DeployedServiceDetails.serviceDeploymentState.DEPLOYMENT_FAILED,
+    const [newServiceId, setNewServiceId] = useState<string | undefined>(undefined);
+    const [oldServiceId, setOldServiceId] = useState<string | undefined>(undefined);
+
+    const migrateServiceDetailsQuery = useMigrateServiceDetailsPollingQuery(migrationId, isMigrateRequestSuccess, [
+        ServiceMigrationDetails.migrationStatus.MIGRATION_COMPLETED,
+        ServiceMigrationDetails.migrationStatus.MIGRATION_FAILED,
+        ServiceMigrationDetails.migrationStatus.DESTROY_FAILED,
+        ServiceMigrationDetails.migrationStatus.DATA_IMPORT_FAILED,
+        ServiceMigrationDetails.migrationStatus.DEPLOY_FAILED,
+        ServiceMigrationDetails.migrationStatus.DATA_EXPORT_FAILED,
     ]);
+    const deployServiceDetailsQuery = useServiceDetailsPollingQuery(newServiceId, serviceHostingType);
```

Comment:
@WangLiNaruto we do not need the polling for service details. We must directly fetch the service details at the end of the migration. 

1. remove the word polling. 
2. line 47 and 48 will make react-query to get details as soon as service IDs are available. Instead, we must call the query manually to get data whenever the migration reaches final state and then set the alert accordingly.